tests/dummy/app/routes/application.ts

```diff
@@ -18,6 +20,17 @@ export default class ApplicationRoute extends Route {
     this.setupTracking();
   }
 
+  @action
+  didTransition(): void {
+    if (
+      config.environment !== 'test' &&
+      window &&
+      typeof window.scrollTo === 'function'
+    ) {
+      window.scrollTo(0, 0);
```

Comment:
We could do `window?.scrollTo?.(0, 0)` instead to get rid of the `typeof` check.

Reply:
This fails in FastBoot - i.e. prember - without this check.